Silk.NET.sln

```diff
@@ -357,6 +357,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Silk.NET.Maths.Tests", "src
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Utf8Test", "src\Lab\Utf8Test\Utf8Test.csproj", "{C06556DC-52AF-44AE-99B8-1251C18C3973}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Silk.NET.GLFW.Tests", "src\Windowing\Silk.NET.GLFW.Tests\Silk.NET.GLFW.Tests.csproj", "{043D78F1-3D65-4439-BDEB-EBA10DE44810}"
```

Comment:
I think we should move all of the tests into a new `tests/` directory. That way they're easily accessible.

Reply:
Good point. Would you also move the maths tests there, or just the other(s)?

Reply:
Probably all of them.

Reply:
In that case I'd prefer to make another PR moving them, just for that sweet git log 😄 
Would that be ok with you?

Reply:
Yeah sure. Will move onto the actual review after a dog walk.